Sming/Components/Partition/src/Arch/Host/base.csv

```diff
@@ -0,0 +1,3 @@
+# Name,   Type, SubType, Offset, Size,    Flags
```

Comment:
This configuration file should go under Sming/Arch/Host as well as the C/C++ files.